repos/system_upgrade/el7toel8/actors/checkbind/libraries/model.py

```diff
@@ -0,0 +1,111 @@
+    else:
+        state[name] = [(stmt_text, statement.config.path)]
+
+
+def find_dnssec_lookaside(statement, state):
+    try:
+        arg = statement.var(1)
+        if not(arg.type() == arg.TYPE_BARE and arg.value() == 'no'):
+            # auto or yes statement
+            # dnssec-lookaside "." trust-anchor "dlv.isc.org";
+            add_statement(statement, state)
+    except IndexError:
+        api.current_logger().warning('Unexpected statement format: "%s"',
+                                     statement.serialize_skip(' '))
+
+
+def convert_to_issues(statements):
+    """Produce list of offending statements in set of files.
+
+    :param statements: one item from list created by add_statement
+    """
+
+    files = dict()
+    for statement, path in statements:
+        if path in files:
+            files[path].update(statement)
+            if statement not in files[path].statements:
+                files[path].statements.append(statement)
+        else:
+            files[path] = set(statement)
+    values = list()
+    for path in files:
+        values.append(BindConfigIssuesModel(path=path, statements=list(files[path])))
+    return values
+
+
+def convert_found_state(state, files):
+    """Convert find state results to facts.
+
+    Check found statements and create facts from them."""
+
+    dnssec_lookaside = None
+    if 'dnssec-lookaside' in state:
+        dnssec_lookaside = convert_to_issues(state['dnssec-lookaside'])
+
+    return BindFacts(config_files=files,
+                     modified_files=[],
+                     dnssec_lookaside=dnssec_lookaside,
+                     listen_on_v6_missing='listen-on-v6' not in state)
+
+
+def get_facts(path, log=None):
+    """Find issues in configuration files.
+
+    Report used configuration files and wrong statements in each file.
+    """
+
+    find_calls = {
+        'dnssec-lookaside': find_dnssec_lookaside,
+        'listen-on-v6': add_statement,
+    }
+
+    parser = isccfg.IscConfigParser(path)
+    state = {}
+    files = set()
+
+    for cfg in parser.FILES_TO_CHECK:
+        parser.walk(cfg.root_section(), find_calls, state)
+        files.add(cfg.path)
+
+        api.current_logger().debug('Found state: "%s", files: "%s"',
+                                   repr(state), files)
+
+    facts = convert_found_state(state, list(files))
+    return facts
+
+
+def make_report(facts):
+    """Make report message from gathered facts."""
+    summary_messages = []
+    report = []
+    if facts.dnssec_lookaside:
+        summary_messages.append('BIND configuration contains no longer accepted statements: dnssec-lookaside.')
+    if facts.listen_on_v6_missing:
+        summary_messages.append('Default value of listen-on-v6 have changed, but it is not present in configuration.'
+                                ' named service will now listen on INET6 sockets also.')
+
+    if summary_messages:
+        summary = ' '.join(summary_messages)
+        report.extend(
```

Comment:
Did not pass final testing:

```
Process Process-371:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib64/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 72, in _do_run
    actor_instance.run(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/leapp/actors/__init__.py", line 290, in run
    self.process(*args)
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/actors/checkbind/actor.py", line 33, in process
    report = model.make_report(facts)
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/actors/checkbind/libraries/model.py", line 108, in make_report
    reporting.Flags([reporting.Flags.INHIBITOR]),
TypeError: extend() takes exactly one argument (5 given)
```

Please provide a fix + add unit test and do not hesitate to try a simple integration test. Thank you.